### ISSUE TITLE
no sky

### ISSUE THREAD

**Issue by m4son** [jdolan/quake2#6]
There is no sky, a map becomes a sky, messy stuff, set sky something doesnt work either. Changing gl_clear 1 makes it red/orange.

**Comment by jdolan** [comment]
Steps to reproduce? What map? Retail pak0, or demo pak0? What operating system?

**Comment by m4son** [comment]
Your installation, q2 demo pak0.pak, anymap map, whatever mod, no weapons/entities seen, LinuxMint 64

**Comment by jdolan** [comment]
Can you attach a screenshot or link to a video showing the problem? Does the problem appear intermittently, or as soon as the map loads? Help me out here, man. Have you posted a bug report before?

**Comment by jdolan** [comment]
Thanks for posting the video:
https://www.dropbox.com/s/xzdmfwhq2uvdml2/quake2_3d-2016-05-07_23.22.31.mp4

So no sky box, and no entity models either. How strange!

It looks like you have the high-resolution HUD pak installed. Did you also install the high-resolution items and weapons pak? If so, can you try renaming it temporarily to see if it makes a difference?

Also, could you try:

    gl_replacemd2 0
    gl_replacepcx 0
    gl_replacewal 0
    vid_restart

And if that doesn't change anything, then please post the output from `cvarlist` here as well.

Thanks!

**Comment by m4son** [comment]
Yes, everything from your install was downloaded. I have removed some paks, generations/extras/hud  and it don't change anything. 

http://pastebin.com/EHviGteH

Also, if i turn off gl_ext_pointparameters i dont even get a weapon shooting trails, basically no particles at all.
